services/autoscaling/src/simcore_service_autoscaling/modules/instrumentation/_models.py

```diff
@@ -36,7 +37,12 @@ def __post_init__(self) -> None:
         cluster_subsystem = f"{self.subsystem}_cluster"
         # Creating and assigning gauges using the field names and the metric definitions
         for field_name, definition in CLUSTER_METRICS_DEFINITIONS.items():
-            gauge = create_gauge(field_name, definition, cluster_subsystem)
+            gauge = create_gauge(
+                field_name=field_name,
+                definition=definition,
+                subsystem=cluster_subsystem,
+                registry=self.registry,
```

Comment:
could you please also do the same for the `director-v2`  all code is here https://github.com/ITISFoundation/osparc-simcore/blob/9057d01382ef51a68e86c39c2ef96bda3c87fbb6/services/director-v2/src/simcore_service_director_v2/modules/instrumentation/_models.py